models/issue.go

```diff
@@ -63,6 +64,7 @@ type Issue struct {
 	Reactions        ReactionList  `xorm:"-"`
 	TotalTrackedTime int64         `xorm:"-"`
 	Assignees        []*User       `xorm:"-"`
+	ProjectIssueID   ProjectIssue
```

Comment:
```suggestion
	ProjectIssueID   ProjectIssue `xorm:"-"`
```

Reply:
@6543 I can't do this because otherwise my query will stop working here https://github.com/patcito/gitea/blob/projects-enhancements/models/project_board.go#L206 as I use a field called project_issue_id in the join query and xorm will remove this field if I apply your suggestion of adding `xorm:"-"`

Reply:
But ID should not be `ProjectIssue` type. Also this will generate columns to `issue` table just like `ExcludeProjectID`